Tasks/CopyFilesV2/task.json

```diff
@@ -16,8 +16,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 190,
-        "Patch": 1
+        "Minor": 197,
```

Comment:
Since we've already have Courtesy Push PR prepared - could it make sense to bump it to 198?

Reply:
Ok, I will bump it to 198